docs/en-US/commands/Remove-JiraIssue.md

```diff
@@ -0,0 +1,181 @@
+---
+external help file: JiraPS-help.xml
+Module Name: JiraPS
+online version:
+schema: 2.0.0
+---
```

Comment:
Missing some metadata. Eg:
```
---
external help file: JiraPS-help.xml
Module Name: JiraPS
online version: https://atlassianps.org/docs/JiraPS/commands/Remove-JiraGroup/
locale: en-US
schema: 2.0.0
layout: documentation
permalink: /docs/JiraPS/commands/Remove-JiraGroup/
---
```

Reply:
Commit 645087886416d89cb09d8183015625350621bc5f added the missing metadata.